src/views/SessionView/index.vue

```diff
@@ -77,6 +77,9 @@ export default {
 
     window.addEventListener("resize", this.handleResize);
   },
+  beforeDestroy() {
+    window.removeEventListener("resize", this.handleResize);
```

Comment:
@jkeat,

Removing the event listener here helps get rid of the "Back to dashboard" header when we're out of the SessionView

<img width="793" alt="ss" src="https://user-images.githubusercontent.com/35900194/78050964-044b3300-734b-11ea-8379-0b915f7c9b16.png">